owlbot-python: post-processor crashing update lock jobs

Example: https://pantheon.corp.google.com/cloud-build/builds;region=global/ac2d4666-ba10-4b63-ba32-814d1083883b;step=9?project=repo-automation-bots

```
nox > Error while collecting sessions.
nox > Sessions not found: blacken
2021-10-06 18:34:29,870 synthtool [ERROR] > Failed executing nox -s blacken:
```

Example: https://pantheon.corp.google.com/cloud-build/builds;region=global/5d623c1a-fe62-459c-a405-676009704bf1;step=9?project=repo-automation-bots

```
Traceback (most recent call last):
  File "/usr/local/bin/nox", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/nox/__main__.py", line 45, in main
    exit_code = workflow.execute(
  File "/usr/local/lib/python3.9/site-packages/nox/workflow.py", line 49, in execute
    return_value = function_(*args, global_config=global_config)
  File "/usr/local/lib/python3.9/site-packages/nox/tasks.py", line 58, in load_nox_module
    check_nox_version(global_config.noxfile)
  File "/usr/local/lib/python3.9/site-packages/nox/_version.py", line 110, in check_nox_version
    needs_version = _read_needs_version(filename)
  File "/usr/local/lib/python3.9/site-packages/nox/_version.py", line 74, in _read_needs_version
    return _parse_needs_version(source, filename=filename)
  File "/usr/local/lib/python3.9/site-packages/nox/_version.py", line 55, in _parse_needs_version
    module: ast.Module = ast.parse(source, filename=filename)
  File "/usr/local/lib/python3.9/ast.py", line 50, in parse
    return compile(source, filename, mode, flags,
  File "/workspace/python-logging/noxfile.py", line 91
    session.install("flask", "-c", constraints_path)session.install("webob", "-c", constraints_path)session.install("django", "-c", constraints_path)
                                                    ^
SyntaxError: invalid syntax
2021-10-06 17:34:09,693 synthtool [ERROR] > Failed executing nox -s blacken:
```

--- a/synthtool/gcp/templates/python_library/noxfile.py.j2
+++ b/synthtool/gcp/templates/python_library/noxfile.py.j2
@@ -88,7 +88,9 @@ def default(session):
         CURRENT_DIRECTORY / "testing" / f"constraints-{session.python}.txt"
     )
     session.install("mock", "asyncmock", "pytest", "pytest-cov", "pytest-asyncio", "-c", constraints_path)
-    {% for d in unit_test_external_dependencies %}session.install("{{d}}", "-c", constraints_path){% endfor %}
+    {%- for d in unit_test_external_dependencies -%}
+    session.install("{{d}}", "-c", constraints_path)
+    {% endfor %}
     {% for dependency in unit_test_local_dependencies %}session.install("-e", "{{dependency}}", "-c", constraints_path)
     {% endfor %}
     {% for dependency in unit_test_dependencies %}session.install("-e", "{{dependency}}", "-c", constraints_path){% endfor %}

